test/es6/rlexe.xml

```diff
@@ -1070,7 +1070,7 @@
   <test>
     <default>
       <files>bug_OS_2553885.js</files>
-      <tags>exclude_win7,BugFix</tags>
+      <tags>exclude_win7,BugFix,exclude_xplat</tags>
```

Comment:
Sure. As already mentioned from #1061 waiting for the investigation for disabled tests.